README.md

```diff
@@ -14,7 +23,7 @@ All commit messages must start with one of the following prefixes.
 * **"test:"** - New tests or correcting existing tests
 * **"chore:"** - Miscellaneous changes
 
-## Examples
+### Examples
```

Comment:
why are there three #s

Reply:
To make it clear that the examples are a sub category  of the prefixes